src/connectors/configure-related-items/connectConfigureRelatedItems.ts

```diff
@@ -130,11 +139,11 @@ See https://www.algolia.com/doc/api-reference/api-parameters/optionalFilters/
       ),
     };
 
-    const makeConfigure = connectConfigure(renderFn, unmountFn);
+    const makeConfigure = connectConfigure<TWidgetParams>(renderFn, unmountFn);
 
     return {
-      ...makeConfigure({ searchParameters } as any),
+      ...makeConfigure({ searchParameters } as TWidgetParams),
```

Comment:
```suggestion
      ...makeConfigure({ searchParameters }),
```

What TS error do you get if you do it just like this?

Reply:
I am going to alias to any here. I propose we re-visit this topic once we analyse if searchParameters should be a type of the connector or widget or wtv.